internal/containerizedengine/types.go

```diff
@@ -67,6 +64,12 @@ var (
 			NoNewPrivileges: false,
 		},
 	}
+
+	// RuntimeMetadataName is the name of the runtime metadata file
+	RuntimeMetadataName = "distribution_based_engine"
```

Comment:
This should be a `const`. It's also not used outside this package, so should not be exported